### DISCUSSION TITLE
Laggy output with Logitech C270

### DISCUSSION THREAD

**Discussion by sillitatti** [Motion-Project/motion#1711] [Q&A]
Hello!

I'm trying to use a Logitech C270 web camera with Motion (4.5.1) but it's very laggy and skipping frames. I seem to get a around 1 fps and with motion detection enabled the movie output is even worse, with gaps of several seconds. If I connect the camera to OBS and select MJPEG, the quality is as expected (steady 30 fps). With YUYV it's also somewhat laggy in OBS. Motion log says it is choosing MJPEG but the output looks a lot like YUYV in OBS, just worse.

Any suggestions on what I could try to improve how Motion handles the device?

**Comment by tosiara** [comment]
Please upload full detailed motion log that contains the issue, and, if possible, a sample video with the "laggy" issue
